### PR TITLE
Make Breadcrumbs environment agnostic

### PR DESCRIPTION
**Pull request by briangregoryholmes**
This PR moves the Breadcrumbs component into `web-common` and makes it environment agnostic, allowing it to be added to the `web-local` Dashboard layout as a header. This is a preliminary step for #4561.

### CODE REVIEW

File: web-common/src/components/navigation/breadcrumbs/BreadcrumbItem.svelte [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  $: nextChild = $page.route.id?.split(regex)[depth + 1];
+</script>
+
+<li class="flex items-center gap-x-2 px-2">
+  <div class="flex flex-row gap-x-1 items-center">
+    {#if selected}
+      <a
+        href={!isCurrentPage && selected.href
+          ? selected.href + nextChild
+          : "#top"}
```
Comment (ericpgreen2) [inline web-common/src/components/navigation/breadcrumbs/BreadcrumbItem.svelte:26]: Confirming, it seems like "#top" is supposed to give the user feedback that they did indeed click the link, even though the link doesn't direct them anywhere?

**Review by ericpgreen2 — APPROVED** [review]
Code looks good, just on the dashboard page:
1. I'm missing the Dashboard breadcrumb
2. Clicking on the Project breadcrumb does not navigate me back to the Project page

image

**Comment by ericpgreen2** [comment]
A couple more UXQA findings (though these might already be on your radar for a follow-on PR):
- The Rill icon overlaps the show/hide sidebar menu (though this won't be an issue when we move the dashboard preview to its own tab)
image
- With the breadcrumbs on local, we should remove the redundant title below the breadcrumbs / above the time controls. And we should move the Edit Metrics & Deploy buttons up into the top bar.
image

File: web-common/src/components/navigation/breadcrumbs/BreadcrumbItem.svelte (hunk 1 above)
Comment (briangregoryholmes) [reply web-common/src/components/navigation/breadcrumbs/BreadcrumbItem.svelte:26]: I think it probably doesn't really matter, but #top is just acknowledging that the element is a link that references the current page as opposed to a fully qualified link that happens to be the same as the page you're on.

**Comment by briangregoryholmes** [comment]
Slipped my mind to add the dashboards level to the `web-admin`. Added back now. Your second comment is on my radar for the next PR.

**Pull request merged**